my-dsl.md

```diff
@@ -3,37 +3,112 @@
 _What is the name of the language? Link the name to its webpage
 (if appropriate)._
 
+Despite putting a non-negligible amount of work into this, it doesn't actually
+look like they named the language.
+
+Here is the github link though: https://findlab.github.io/2020/11/15/music-layout-language/
```

Comment:
I don't know anything about sheet music, but this seems like a really helpful DSL. It is very simple and seems easy to learn even for people not experienced in programming.